src/components/MessageBubble.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../styles/theme.style.js';
+import { StyleSheet, Text, View, Image, TouchableOpacity, TextInputComponent, } from 'react-native';
+import TextInputContainer from '../containers/TextInputContainer';
+import { TextInput } from 'react-native-gesture-handler';
+
+
+class MessageBubble extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        isAuthor: true,
+        isSelectable: true,
+      };
+    }
+  
+    render() {
+      return (
+        <MessageBubbleContainer isAuthor={this.state.isAuthor} backgroundColor={this.props.labelColor} isSelectable={this.props.isSelectable}>
```

Comment:
you can move isSelectable={this.props.isSelectable} in WrittenMessage (after line 23) and change it to editable={this.props.isSelectable} to get the expected behaviour.

Reply:
Use this.props.isAuthor instead of this.state.isAuthor, as the boolean will be passed from the parent (i.e. Conversation screen where we will be mapping the message bubbles)